tests/system_health_test.py

```diff
@@ -0,0 +1,329 @@
+#! /usr/bin/python -u
+
+import sys
+import os
+import click
+from click.testing import CliRunner
+import mock_tables.dbconnector
```

Comment:
Organize imports per PEP8 standards:

```python
import os
import sys

import click
import mock_tables.dbconnector
from click.testing import CliRunner
```

Reply:
@jleveque Fixed.

Reply:
Please add a line break between `import sys` and `import click` as in my example to separate standard lib imports from third-party imports.